shared/constants/chat2/index.tsx

```diff
@@ -171,6 +171,34 @@ export const getMessage = (
   id: Types.ConversationIDKey,
   ordinal: Types.Ordinal
 ): Types.Message | null => state.chat2.messageMap.getIn([id, ordinal])
+export const getDecoratedMessage = (
```

Comment:
likely better to just make this a type guard instead of this message or null function
https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards